config/initializers/carrier_wave.rb

```diff
@@ -1,16 +1,18 @@
 CarrierWave.configure do |config|
   if Rails.env.staging? || Rails.env.production?
     config.storage    = :aws
-    config.aws_bucket = ENV.fetch('S3_BUCKET_NAME')
+    config.aws_bucket = ENV['S3_BUCKET_NAME']
     config.aws_acl    = 'public-read'
+    config.aws_authenticated_url_expiration = 60 * 60 * 24 * 5
 
     config.aws_credentials = {
-      access_key_id:     ENV.fetch('AWS_ACCESS_KEY_ID'),
-      secret_access_key: ENV.fetch('AWS_SECRET_ACCESS_KEY'),
-      region:            ENV.fetch('AWS_REGION') # Required
```

Comment:
The `.fetch` was fine or is fine, because if the variable is not set - you get an exception.